src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReader {
```

Comment:
```suggestion
public interface FileReaderService {
```
В jav.io уже есть класс  FileReader, лучше переименовать, что бы было понятнее что именно вызываешь и небыло наложений имен...

Reply:
Agree. Done.